lib/analytics-page/addon/application/controller.ts

```diff
@@ -70,10 +45,10 @@ export default class ApplicationController extends Controller {
     @reads('node.apiMeta.templated_by_count')
     templatedByCount?: number;
 
-    @computed('node.public', 'model.{id,modelName}')
+    @computed('node.public')
     get nodePublic() {
-        const node: Node | null = this.node || this.store.peekRecord(this.model.modelName, this.model.id);
-        return node && node.public;
+        const { node } = this;
+        return !node || node.public;
```

Comment:
I'm not really understanding the `!node` here. Why is node publicness considered true if node is undefined or null?

Reply:
good catch, cleaned up